InterfaceParser/parsers/json_rpc.py

```diff
@@ -0,0 +1,98 @@
+"""JSON RPC parser.
+
+Contains parser for JSON RPC XML format.
+
+"""
+from parsers.parse_error import ParseError
+from parsers.rpc_base import RPCBase
+
+
+class Parser(RPCBase):
+    """JSON RPC parser."""
+
+    def __init__(self):
+        """Constructor."""
+        super(Parser, self).__init__()
+        self._interface_name = None
+
+    @property
+    def get_version(self):
+        return '1.0.0'
+
+    def _parse_root(self, root):
+        """Parse root XML element.
+
+        This implementation parses root as interfaces element with multiple
+        interfaces in it.
+
+        :param root: ElementTree root Element object
+        """
+
+        self._params = root.attrib
+        self._interface_name = None
+
+        for element in root:
+            if element.tag != "interface":
+                raise ParseError("Subelement '{}' is unexpected in interfaces".format(element.tag))
+
+            if "name" not in element.attrib:
+                raise ParseError("Name is not specified for interface")
+
+            self._interface_name = element.attrib["name"]
+            self._parse_interface(element, self._interface_name + "_")
+
+    def _check_enum_name(self, enum):
+        """ Check enum name.
+
+        This method is called to check whether the newly parsed enum's name
+        conflicts with some predefined enum.
+        As SDLRPCV2 has no predefined enums this implementation does nothing.
+
+        :param enum: an instance of model.Enum
+        """
+
+    def _provide_enum_element_for_function(self, enum_name, element_name):
+        """Provide enum element for functions.
+
+        This implementation replaces the underscore separating interface and
```

Comment:
Sorry I do not understand what this description is trying to say. 

Maybe

> This method returns an enum elements name and internal name (if exists). If the enum is a FunctionID, the '_' in the name will be replace with a "."

Reply:
First of all I want to mention that this class with all descriptions came from legacy sdl_core.
As for me the description is pretty clear, if enum_name equal to "FunctionID" in element_name (string) underscore (_) will be replaced with dot (.).
Otherwise you can provide description which is more convenient from your perspective.